### DISCUSSION TITLE
The number of energy bands increased when symmetrizing Wannier Hamiltonian of Te

### DISCUSSION THREAD

**Discussion by LHZ-HNU** [wannier-berri/wannier-berri#340] [Q&A]
![symmetrized (12)](https://github.com/user-attachments/assets/1a78dd03-e383-42fa-9698-e0625a81c325)
![symmetrized (13)](https://github.com/user-attachments/assets/b1b4fdee-9920-42c2-bf58-ca75c43d0399)
[Te.zip](https://github.com/user-attachments/files/16310529/Te.zip)
Here are my POSCAR, wannier90.win, wannier90_tb, and .py files.
I don't understand why this is happening.
Thank you all for your answers.

**Comment by stepan-tsirkin** [comment]
your zip does not include .py file, so cannot say for sure

However, the symmetrization procedure works reliably only when maximal localization is NOT performed, i.e. num_iter=0

Also, note (#338)  vasp changes the order of spins in projection in latest version, therefeore you need to specify "espresso"  as a DFT code.

**Comment by LHZ-HNU** [comment]
Thank you for your reply amidst your busy schedule sir！
When calculating Te in the example, I set num_inter=0 and the orbits in the projector and wannier90.win is consistent. However, there were still obvious errors after the symmetrization process. May I ask what the reason is? The software version is vasp5.4.4+wannier90-2.1. Some files are as follows. 
Thank you very much, and wish you a nice day!
[Te.zip](https://github.com/user-attachments/files/16758615/Te.zip)
![symmetrized (4)](https://github.com/user-attachments/assets/aa3496a9-0f9b-4965-be4c-b3c747024409)
![unsymmetrized (5)](https://github.com/user-attachments/assets/c13f413f-fa9e-4c68-8eab-fea66e252781)

**Comment by stepan-tsirkin** [comment]
In your win file num_iter=10000

![image](https://github.com/user-attachments/assets/98ecf16b-e2d1-432c-9fb5-5cfd9275201d)

It is likely that wannier90 hybridised into sp3 instead of s and p. Therefore, the symmetrization went totally wrong.

To exclude the spin-projection-order issue, try to achieve good results first in a scalar calculation

Also, it is generally not recommended to use _tb.dat files, better to read the .chk (Systrem_w90) , however this is not the big issue here. 

To get a more meaningful help. please provide your python file and the .wout file (to see how the wannierisation ended)

**Reply by LHZ-HNU** [reply]
Thank you for your reply and help.

I have seted num_iter=0 in the new wannier90.win,but there were still obvious errors of bands after the symmetrization process . 

The python file and the .wout file are provided as py.out and wannier90.wout.

[Te3.zip](https://github.com/user-attachments/files/16780645/Te3.zip)

**Reply by stepan-tsirkin** [reply]
According tothe spreads in  your wout file, first there are 6 projections of one type (I guess, they are 's') and then 18 projections of other type (I guess t1
they are 'p') 
This looks like QE style, olf VASP would put first 12 projections for spin-up, then 12 projections with spin-down.

 so use `DFT_code='qe'` , did you try that? 

![image](https://github.com/user-attachments/assets/86cc2f1b-aaf8-4bab-9125-1f6cba677dc7)

**Reply by LHZ-HNU** [reply]
By using DFT_code='qe', I got the right bands. I offer my most sincere thanks.

![unsymmetrized (6)](https://github.com/user-attachments/assets/ce3a4a0c-d555-45a4-be2c-66e8073f80ee)
![symmetrized (11)](https://github.com/user-attachments/assets/d1109559-99bd-48a9-bab7-98f149cb0bfa)
